jaeger-spark-dependencies-common/src/main/java/io/jaegertracing/spark/dependencies/SpansToDependencyLinks.java

```diff
@@ -143,14 +142,14 @@ static boolean isServerSpan(Span span) {
         return dependencies;
     }
 
-    private Optional<Dependency> sharedSpanDependency(Set<Span> sharedSpans) {
+    protected Optional<Dependency> sharedSpanDependency(Set<Span> sharedSpans) {
```

Comment:
Why changing visibility?

Reply:
To make it testable otherwise it should be moved to another class, i wanted to keep ur code structure